.github/workflows/python-package.yml

```diff
@@ -1,13 +1,10 @@
 # This workflow will install Python dependencies, run tests and lint with a variety of Python versions
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
 
-name: Python package
 
-on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+name: DoWhy CI
+
+on: [push, pull_request]
```

Comment:
You've removed the restriction to a specific branch (now `main`). What's the idea behind it? Do you think that's more appropriate? Asking, because I'm not sure we'd want to run this on every push to an arbitrary branch.

Reply:
That's fair - we use this terser form on some other open-source projects but I don't mind restoring the branch selector.